### PR TITLE
Feature/update section

### PR DESCRIPTION
**Pull request by greeshmareddyg**
Created Updates Section:

https://github.ibm.com/brand/stories/issues/1461

Screen Shot 2019-10-29 at 12 07 26 PM 2 2 2

Screen Shot 2019-10-29 at 12 09 30 PM 2

### CODE REVIEW

File: src/pages/updates/road-map.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Roadmap
+description: The IBM grid is the framework for all the visual elements and typography. The grid gives you structure and guidance so you can focus on your creative idea.
```
Comment (jnm2377) [inline src/pages/updates/road-map.mdx]: Can we update the description for this? Looks like it was copied from the grid page.

File: src/pages/updates/road-map.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Expected in Q1 2020
+
+## IBM Plex Korean (Hangul)
+
+For the last year we’ve been working on the CJK (Chinese Japanese Korean) version of IBM Plex and we’re expecting to release IBM Plex Sans Korean in Q1 2020. We’re excited about the whole set and look forward to making this available in the new year!
+
+Expected in Q1 2020
+
+## Data Visualization
```
Comment (alisonjoseph) [inline src/pages/updates/road-map.mdx]:
```suggestion
## Data visualization
```

File: src/pages/updates/whats-new.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+<Row>
+<Column colMd={6} colLg={8}>
+
+![Exporting stroke app icons: step 3](/images/color.svg)
+
+</Column>
+</Row>
+
+Last updated on 25 september 2019 <br/>
```
Comment (alisonjoseph) [inline src/pages/updates/whats-new.mdx]:
```suggestion
Last updated on 25 September 2019 <br/>
```

File: src/pages/updates/whats-new.mdx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+[Find the details on our color page in IDL ](https://github.com/carbon-design-system/carbon/raw/master/packages/colors/artifacts/IBM_Colors.zip)
```
Comment (alisonjoseph) [inline src/pages/updates/whats-new.mdx]: Is this the correct text/link? Seems out of place and doesn't link to the color page.

Comment (greeshmareddyg) [reply src/pages/updates/whats-new.mdx]: Updated the link to color page.

**Comment by alisonjoseph** [comment]
Needs a design reviewer

**Comment by greeshmareddyg** [comment]
@jnm2377 @sadek could please review the PR.

**Pull request merged**